File: Chapter 05/C Code/5_8.c
#include <stdio.h>

int main() {
    int matrix[3][3];
    int i, j;
    
    // Input matrix
    printf("Enter the elements of the matrix:\n");
    for (i = 0; i < 3; i++) {
        for (j = 0; j < 3; j++) {
            scanf("%d", &matrix[i][j]);
        }
    }
    
    // Print matrix
    printf("The matrix you entered is:\n");
    for (i = 0; i < 3; i++) {
        for (j = 0; j < 3; j++) {
            printf("%d ", matrix[i][j]);
        }
        printf("\n");
    }
    
    return 0;
}

File: Chapter 05/Python code/5_8.py

matrix = [[0 for j in range(3)] for i in range(3)]

for i in range(3):
    for j in range(3):
        matrix[i][j] = int(input("Enter element at position ({,i,},{,j, }): "))

print("The matrix is:")
for i in range(3):
    for j in range(3):
        print(matrix[i][j], end=" ")
    print()

